src/AbstractFunctionReferenceFixer.php

```diff
@@ -67,7 +67,7 @@ protected function find($functionNameToSearch, Tokens $tokens, $start = 0, $end
         $matches = $tokens->findSequence($candidateSequence, $start, $end, false);
         if (null === $matches) {
             // not found, simply return without further attempts
-            return;
+            return null;
```

Comment:
It's the same thing coming out of the function in practice. the void type doesn't actually exist, it's just used by the type checker in php 7.1 to assert all return statements have no statement following them.